verilog/tools/formatter/verilog_format.cc

```diff
@@ -216,22 +194,62 @@ int main(int argc, char** argv) {
     }
     if (absl::GetFlag(FLAGS_failsafe_success)) {
       // original text was preserved, and --inplace modification is skipped.
-      return 0;
+      return true;
     }
-    return 1;
+    return false;
   }
 
   // Safe to write out result, having passed above verification.
   if (inplace && !is_stdin) {
-    if (!verible::file::SetContents(filename, formatted_output)) {
-      std::cerr << "Error writing to file: " << filename << std::endl;
-      std::cerr << "Printing to stdout instead." << std::endl;
-      std::cout << formatted_output;
-      return 1;
+    // Don't write if the output is exactly as the input, so that we don't mess
+    // with tools that look for timestamp changes (such as make).
```

Comment:
In this case can we print a message to stderr or stdout saying that file was already properly formatted?
I think in multi-file inplace mode it would be good to have file-by-file confirmation as a mark of progress, in case the formatter fails/terminates/crashes on one of the inputs.  Then we at least know the last successful file for triage.

Reply:
Done.